src/kOS/Function/KAC.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
```

Comment:
Well, I was torn between this folder and KOS addon folder, but settled on keeping it here. But I dont have any preference.